runtime/binding-http/src/main/java/io/aklivity/zilla/runtime/binding/http/internal/codec/Http2SettingsFW.java

```diff
@@ -239,6 +240,13 @@ public Builder maxHeaderListSize(long size)
             return this;
         }
 
+        public Builder enableConnectProtocol()
```

Comment:
Suggest renaming this to `connectProtocol`, in keeping with setting method naming for other settings.

Reply:
But the other method names match with their setting names (e.g method `enablePush` for `SETTINGS_ENABLE_PUSH`, and method `maxConcurrentStreams` for `SETTINGS_MAX_CONCURRENT_STREAMS`). In this case the setting name is `SETTINGS_ENABLE_CONNECT_PROTOCOL`.

Reply:
Ah yes, I had missed that - good point, please leave it as-is. 👍